engines/group/app/models/group/group.rb

```diff
@@ -6,11 +6,11 @@ class Group < ActiveRecord::Base
     has_many :memberships, dependent: :destroy
     has_many :users, through: :memberships
     has_many :admins,
-      -> { where "memberships.role = #{ Membership::ROLES[:admin] }" },
+      -> { where "memberships.role = #{Role.new(:admin).index}" },
     through: :memberships,
       source: :user
     has_many :waiters,
-      -> { where "memberships.role = #{ Membership::ROLES[:waiting] }" },
+      -> { where "memberships.role = #{Role.new(:waiters).index}" },
```

Comment:
So maybe is better remove the waiting thing until implemented. 
Maybe in the future we find a better approach.

To me this smell like `externals` because you're creating the relation between that user and the group when there is no relation yet.